salt/orch/update-etc-hosts.sls

```diff
@@ -0,0 +1,23 @@
+update_grains:
+  salt.function:
+    - tgt: 'roles:kube-(master|minion)'
+    - tgt_type: grain_pcre
+    - name: saltutil.refresh_grains
+
+update_mine:
+  salt.function:
+    - tgt: 'roles:kube-(master|minion)'
+    - tgt_type: grain_pcre
+    - name: mine.update
+    - require:
+      - salt: update_grains
+
+etc_hosts_setup:
+  salt.state:
+    - tgt: 'roles:kube-(master|minion)'
+    - tgt_type: grain_pcre
+    - concurrent: True
+    - sls:
+      - etc-hosts
```

Comment:
we also need to add the `kube-apiserver` sls here - https://github.com/kubic-project/salt/blob/f119228e4fad8986b87ed3a932ae19817c087213/salt/kubernetes-master/apiserver.jinja#L24